File: pharmacy/src/main/java/isa/pharmacy/Models/Med.java
package isa.pharmacy.Models;

import org.springframework.beans.factory.annotation.Autowired;

public class Med {
    private String code;
    private String name;
    private String type;
    private Form form;
    private String [] ingredients;
    private String producer;
    private String [] replacementCode;
    private IssuanceRegime issuanceRegime; //rezim izdavanja
    private String additionalNotes;

    @Autowired
    private Pharmacy pharmacy;


}

File: client/src/app/dto/med.ts
import { Form } from "./form";
import { IssuanceRegime } from "./issuanceRegime";

export class Med{
    id: Number;
    name: String;
    type: String;
    form: Form;
    producer: String;
    issuanceRegime: IssuanceRegime;
    additionalNotes: String;
    price: Number;
}